app/overrides/hostgroup_play_roles.rb

```diff
@@ -0,0 +1,16 @@
+# Displays Ansible roles tab on Hostgroup form
+Deface::Override.new(
+  :virtual_path => 'hostgroups/index',
+  :name => 'ansible_row_title',
+  :insert_after => "erb[loud]:contains('Actions')",
+  :partial => 'foreman_ansible/ansible_roles/ansible_row_title',
+  :original => '016f2f86bd199275e7f3bebef77fefa90dd218f1'
```

Comment:
Ok, I will remove them, but it produces as a warning in the console: 

```
Deface: [WARNING] No :original defined for 'ansible_row_title', you should change its definition to include:
16:36:28 rails.1   |  |  :original => '016f2f86bd199275e7f3bebef77fefa90dd218f1'
```

I thought that it's useful if there was an alert upon the slightest change in the view because the override could ruin the layout.